backword-compatibility/class-adst-plugins-stats-api-bcp.php

```diff
@@ -0,0 +1,631 @@
+<?php
+/**
+ * Calling W.ORG API Response.
+ *
+ * @package WP Themes & Plugins Stats
+ * @author Brainstorm Force
+ */
+
+/**
+ * Exit if accessed directly.
+ */
+if ( ! defined( 'ABSPATH' ) ) {
+	exit();
+}
+
+/**
+ * Helper class for the ADST Plugin Stats API.
+ *
+ * @since 1.0.0
+ */
+class ADST_Plugins_Stats_Api_Bcp {
```

Comment:
This class name is very long. As it adds shortcodes. You can keep it short to **Shortcodes** 
You can use the namespace to avoid conflict with other classes.